eras/mary/impl/cddl-files/mary.cddl

```diff
@@ -34,10 +38,10 @@ vrf_vkey = bytes .size 32
 vrf_cert = [bytes, bytes .size 80]
 
 operational_cert = 
-  ( hot_vkey        : kes_vkey 
-  , sequence_number : uint     
-  , kes_period      : uint     
-  , sigma           : signature
+  ( hot_vkey        : kes_vkey    
+  , sequence_number : uint .size 8
+  , kes_period      : uint        
```

Comment:
No need to do it in this PR, but could you please make a note to fix `kes_period` to be `uint .size 8` as well.